SignupDirector/SignupDirectorAPI/migrations/0003_participants_attendance.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 4.1.7 on 2023-02-18 10:01
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('SignupDirectorAPI', '0002_participants_name_alter_participants_accompanions_and_more'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='participants',
+            name='attendance',
+            field=models.BooleanField(default=False),
```

Comment:
I'd suggest to combine all migrations in the same PR into one

Reply:
done